build-system/global-configs/experiments-config.json

```diff
@@ -1,5 +1,12 @@
 {
-  "experimentA": {},
+  "experimentA": {
+    "name": "MOVE_FIXED_LAYER",
+    "environment": "AMP",
+    "command": "gulp dist --define_experiment_constant=MOVE_FIXED_LAYER",
+    "issue": "",
+    "expirationDateUTC": "2020-12-31",
+    "define_experiment_constant": "MOVE_FIXED_LAYER"
```

Comment:
was this in camelCase previously? if so, i'll need to change the internal build code as well (minor change, i just want to confirm)

Reply:
@erwinmombay yeah that is correct. @rsimha requested the change. Per his comment I will also change expirationDateUTC to expiration_date_utc

Reply:
ok, that's good to know. as long as i can get that change in before next tuesday this change should be safe. I'll send a CL here shortly internally.

in saying that I think we need more docs on the RTV experiments
/cc @zhouyx

Reply:
Hmm. I certainly didn’t mean to make this difficult. What if you only changed the `argv` stuff and left everything else as is? If that breaks as well, feel free to disregard everything I suggested :)

Reply:
Latest version looks good. Thanks @erwinmombay.